### ISSUE TITLE
Transformers documentation translation to French

### ISSUE THREAD

**Issue by omarespejel** [huggingface/transformers#18322]
Hi!

Let's bring the documentation to all the French-speaking community :)

Who would want to translate? Please follow the 🤗 [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md). Here is a list of the files ready for translation. Let us know in this issue if you'd like to translate any and we'll add your name to the list.

Some notes:

- Please translate using an informal tone (imagine you are talking with a friend about transformers 🤗).
- Please translate in a gender-neutral way.
- Add your translations to the folder called fr inside the [source folder](https://github.com/huggingface/transformers/tree/main/docs/source).
- Register your translation in [fr/_toctree.yml](https://github.com/huggingface/transformers/blob/main/docs/source/it/_toctree.yml); please follow the order of the [English version](https://github.com/huggingface/transformers/blob/main/docs/source/en/_toctree.yml).
- Once you're finished, open a pull request and tag this issue by including #issue-number in the description, where issue-number is the number of this issue. Please ping @omarespejel and @sgugger for review.
- 🙋 If you'd like others to help you with the translation, you can also post in the 🤗 [forums](https://discuss.huggingface.co/).

## Get Started section
- [ ] [index.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/index.mdx) 
- [ ] [quicktour.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/quicktour.mdx).
- [ ] [installation.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/installation.mdx). 

## Tutorial section
- [ ] [pipeline_tutorial.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/pipeline_tutorial.mdx) 
- [ ] [autoclass_tutorial.mdx](https://github.com/huggingface/transformers/blob/master/docs/source/autoclass_tutorial.mdx) 
- [ ] [preprocessing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/preprocessing.mdx) 
- [ ] [training.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/training.mdx) 
- [ ] [accelerate.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/accelerate.mdx) 
- [ ] [model_sharing.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/model_sharing.mdx) 
- [ ] [multilingual.mdx](https://github.com/huggingface/transformers/blob/main/docs/source/en/multilingual.mdx)

**Comment by fffiloni** [comment]
I'd love to help you for this tasks !

**Comment by sgugger** [comment]
Yes, this is something we should do more for the English doc as well. The only exception in my opinion is when something is clearly left to the user like "You need to adapt this constant to your username."

**Comment by fffiloni** [comment]
> Yes, this is something we should do more for the English doc as well. The only exception in my opinion is when something is clearly left to the user like "You need to adapt this constant to your username."

Le Site du Zéro à l'époque le faisait très bien avec parcimonie, c'était très agréable à suivre ;)
Les tutos sont archivés ici: http://sdz.tdct.org/

**Comment by osanseviero** [comment]
Great idea for consistently using 1st person plural!

If possible let's keep the discussion on the issues in English as some of the reviewers might not be French speakers even if we help with making sure the structure/format looks good! :D

**Comment by omarespejel** [comment]
Looks good, is writing in the first person of the plural something we would like to add to the [TRANSLATING guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md), and the description of this and other translation issues?

**Comment by martinez-hugo** [comment]
Hi ! Can I work on this translation ?

**Comment by NinoRisteski** [comment]
Hi, I can do a Macedonian translation if that's an option for contribution?

**Comment by nlegendree** [comment]
Hi, I would like to work on this issue as a French person. It would be my first contribution to this repo. Which files are available to translate in French ?

**Comment by quentin-lauret** [comment]
Hi! I'd like to take the training chapter (docs/source/en/training.md → docs/source/fr/training.md). I have a translation ready and will open a PR shortly.

Note that the French _toctree.yml already reserves a slot for it (local: in_translation / title: Fine-tune un modèle pré-entraîné), so this just fills in an existing placeholder. I've followed the terminology already used in the existing French pages and the conventions agreed in #21456.

cc @stevhliu

**Comment by adam2765** [comment]
Hi! I'd like to take the model sharing page (docs/source/en/model_sharing.md → docs/source/fr/model_sharing.md). The French _toctree.yml already reserves a slot for it (local: in_translation / title: Partager un modèle), so this fills in an existing placeholder. I'll follow the terminology used in the existing French pages and the conventions from #21456, and open a PR shortly.
cc @stevhliu
